src/Sylius/Bundle/ResourceBundle/composer.json

```diff
@@ -28,19 +28,21 @@
         "symfony/form":                    "~2.3",
         "sylius/resource":                 "0.14.*@dev",
         "sylius/storage":                  "0.14.*@dev",
-        "sylius/translation-bundle":       "0.14.*@dev",
         "friendsofsymfony/rest-bundle":    "1.5.*@dev",
         "jms/serializer-bundle":           "~0.13",
         "white-october/pagerfanta-bundle": "~1.0",
         "doctrine/doctrine-bundle":        "~1.3",
-        "willdurand/hateoas-bundle":       "@stable"
+        "willdurand/hateoas-bundle":       "@stable",
+        "stof/doctrine-extensions-bundle": "~1.1"
```

Comment:
Ok, just saw that this comes from translation changes about slug handler... But I guess this could be in `dev` not in main requirements...